barretenberg/cpp/src/barretenberg/bb/main.cpp

```diff
@@ -103,16 +89,14 @@ bool proveAndVerify(const std::string& bytecodePath, const std::string& witnessP
     auto constraint_system = get_constraint_system(bytecodePath);
     auto witness = get_witness(witnessPath);
 
-    auto acir_composer = init(constraint_system);
+    acir_proofs::AcirComposer acir_composer{ 0, verbose };
+    acir_composer.create_circuit(constraint_system, witness);
 
-    Timer pk_timer;
-    acir_composer.init_proving_key(constraint_system);
-    write_benchmark("pk_construction_time", pk_timer.milliseconds(), "acir_test", current_dir);
```

Comment:
what happened to these?

Reply:
all missing benchmarks have been reinstated. my bad